deployment/ansible/roles/model-my-watershed.app/templates/nginx-app.conf.j2

```diff
@@ -1,3 +1,10 @@
+# Protects against scenarios where DNS records for the
+# site being proxied change.
+resolver 8.8.8.8 valid=300s;
```

Comment:
Not sure, but it's used in quite a lot of examples on setting up a `resolver` when proxying to a remote public server.